src/utils/json.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+type JSONValue = Array<JSONValue> | string | number | JSON;
+type JSON = { [key: string]: JSONValue };
+
+// this will mutate the original JSON object
+export function normalize(node: JSON) {
```

Comment:
This utility name sounds very generic but it's not very clear what this does.

What is normalized JSON looking like?
Is it useful in any other places than this test infra?